client/src/Components/Resuable Components/Header/NotLoggedInNavigationBar.js

```diff
@@ -26,9 +28,11 @@ class NotLoggedInNavigationBar extends Component {
   handleItemClick = (e, { name }) => {
     var newUrl;
     if (name == "Write About a Book") {
-      newUrl = "/writeAboutABook";
+      newUrl = "login";
```

Comment:
just wondering why is the newurl being login here instead of /writeaboutabook?

Reply:
because they're not logged in yet, so they can't write about a book

Reply:
👍